package.json

```diff
@@ -20,14 +20,15 @@
     "aliasify": "1.9.0",
     "babel-cli": "6.26.0",
     "babel-eslint": "8.2.2",
-    "babel-jest": "22.4.1",
+    "babel-jest": "^22.4.3",
```

Comment:
was trying to keep these hardcoded to an explicit value to try and prevent unexpected version changes - what do you think?